its/projects/CoverageTest.MultipleProjects/FirstProject/FirstClass.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Linq; // unused
```

Comment:
I've changed that to my favorite issue that is easy to raise and understand :)
```
// ToDo: This should raise S1135
```

Reply:
The issue comments are rather useless in `CoverageTest.MultipleProjects` because we don't assert them. 

I'll leave those comments as they are, just to mention that those pieces of code are intentionally weird. 
I can't change the code, because the coverage report would go out of sync.